crates/biome_js_analyze/src/react/hooks.rs

```diff
@@ -158,7 +158,8 @@ pub(crate) fn is_react_hook_call(call: &JsCallExpression) -> bool {
         .and_then(|ident| ident.name().ok())
         .and_then(|name| name.value_token().ok())
     {
-        if expr.text_trimmed() == "jest" {
+        let exprTrimmed = expr.text_trimmed();
+        if exprTrimmed == "jest" || exprTrimmed == "vi" {
```

Comment:
Please follow the Rust's name convention.

```suggestion
        let expr_trimmed = expr.text_trimmed();
        if expr_trimmed == "jest" || expr_trimmed == "vi" {
```

Or, you can choose another name 😉

```suggestion
        let object_name = expr.text_trimmed();
        if object_name == "jest" || object_name == "vi" {
```